packages/aws-cdk-lib/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -216,7 +218,7 @@ export class TokenAuthorizer extends LambdaAuthorizer {
       authorizerUri: lambdaAuthorizerArn(props.handler),
       authorizerCredentials: props.assumeRole?.roleArn,
       authorizerResultTtlInSeconds: props.resultsCacheTtl?.toSeconds() ?? Duration.minutes(5).toSeconds(),
-      identitySource: props.identitySource || 'method.request.header.Authorization',
+      identitySource: props.identitySource || IdentitySource.header('Authorization'),
```

Comment:
The change caused by this is guaranteed by [this test](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-apigateway/test/authorizers/lambda.test.ts#L30).